src/pages/StatisticsChart/CellCount.tsx

```diff
@@ -88,6 +88,7 @@ const getOption = (statisticCellCounts: State.StatisticCellCount[], isThumbnail
       {
         name: i18n.t('statistic.live_cell'),
         type: 'line',
+        stack: '总量',
```

Comment:
Is the chart displayed only in Chinese?

Reply:
The Echarts example gave me a mistake and this value is just be same between series of chart option.

I updated to `cell_count` to understand more easily. https://github.com/nervosnetwork/ckb-explorer-frontend/pull/428/commits/da9b61494adb8832a7e67fa020f3be7bbb18eb53